pkg/kubelet/kubelet.go

```diff
@@ -2074,7 +2073,7 @@ func (kl *Kubelet) canAdmitPod(pods []*api.Pod, pod *api.Pod) (bool, string, str
 func (kl *Kubelet) syncLoop(updates <-chan kubetypes.PodUpdate, handler SyncHandler) {
 	glog.Info("Starting kubelet main sync loop.")
 	kl.resyncTicker = time.NewTicker(kl.resyncInterval)
```

Comment:
I will do it later since it changes more tests, and I'll have to rebase my other PR again :)